### PR TITLE
Add short note on Windows Tips

### PR DESCRIPTION
**Pull request by DoumanAsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -378,6 +378,42 @@ $ cargo test
...
+
+But powershell function aliases do not behave like your typical linux shell alias.
+
+You always need to propagate arguments and **Stdin** input.
+But it cannot be done simply as `function grep() { $input | rg.exe --hidden $args }`
+
+Use below example as reference to how setup alias in powershell.
+
+```powershell
+function grep {
```
Comment (BurntSushi) [inline README.md]: I think this looks OK to me. Is there any way to put this function into a file that is automatically loaded whenever powershell starts?

cc @retep998

Comment (DoumanAsh) [inline README.md]: Ah yes, i add notes on that too

**Comment by DoumanAsh** [comment]
I added note on how to use Powershell Profile

File: README.md [hunk 2]
```diff
@@ -378,6 +378,53 @@ $ cargo test
 
 from the repository root.
 
+### Tips
+
+#### Windows Powershell
+
+##### Powershell Profile
+
+To customize powershell on start-up there is a special powershell script that is needed to be crated.
```
Comment (retep998) [inline README.md]: crated?

File: README.md [hunk 3]
```diff
@@ -378,6 +378,53 @@ $ cargo test
...
+In order to find its location run command `Get-Command $profile | Select-Object -ExpandProperty Definition`
+
+But it's typical location should be following: `C:\Users\<your_user_name>\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1`
```
Comment (retep998) [inline README.md]: `it's` -> `its`

File: README.md (hunk 2 above)
Comment (DoumanAsh) [inline README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -378,6 +378,53 @@ $ cargo test
...
+#### Windows Powershell
+
+##### Powershell Profile
+
+To customize powershell on start-up there is a special powershell script that has to be created.
+In order to find its location run command `Get-Command $profile | Select-Object -ExpandProperty Definition`
+
+But its typical location should be following: `C:\Users\<your_user_name>\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1`
+If there is no directory `WindowsPowerShell` then it must be created too.
+
```
Comment (elirnm) [inline README.md]: The location of the profile can be gotten just with `$profile`; no need for that complex command. And you can check to see if a profile exists with `test-path $profile`. You might want to link to the [official docs on profiles](https://technet.microsoft.com/en-us/library/bb613488(v=vs.85).aspx), which cover the various profile locations and have instructions on how to create one if it doesn't exist (`new-item -path $profile -itemtype file -force`).

Comment (DoumanAsh) [inline README.md]: Well, ok...

**Comment by BurntSushi** [comment]
@DoumanAsh Thanks again!

**Pull request merged**